application/forms/ScheduleForm.php

```diff
@@ -151,9 +154,22 @@ public function removeSchedule(int $id): void
 
     protected function assemble()
     {
+        if (! $this->showRemoveButton) {
+            $this->addHtml(new HtmlElement(
+                'p',
+                new Attributes(['class' => 'description']),
+                new Text($this->translate(
+                    'Organize contacts and contact groups in a time based schedule and let them rotate'
+                    . ' automatically. Multiple rotations can be added to your schedule to represent the rotating'
+                    . ' members in your admin team. Schedules can be used as recipients of event rules.'
```

Comment:
This poses another question for me :rofl: : What is an *admin team*?

I think a less specific but still not too abstract example is required, to explain why multiple rotations are useful.

Something along this line:

*Multiple rotations allow differing rotation patterns to override others.*

But not quite this line, I just cannot come up with a better alternative right now. But I hope you get the idea.